docs/installation.rst

```diff
@@ -39,8 +39,14 @@ environments should consider using other packages that help manage them such as
 `pyenv-virtualenvwrapper <https://github.com/yyuu/pyenv-virtualenvwrapper>`_.
 
 
+Sources
+-------
+
+Girder can be installed from two sources, **pip** or **git repository**.
```

Comment:
Both of these actually should be installed using pip, the real distinction is between **pypi** and **git**.